### DISCUSSION TITLE
Option to show the Bundle-Symbolic-Name in the reactor summary instead of the bundle name

### DISCUSSION THREAD

**Discussion by vogella** [eclipse-tycho/tycho#455] [Ideas]
I just debugged a large Tycho build for a client. It was harder than in the past to find a failing module as the Tycho build uses the plug-in name and not the bundle-symbolic-name in its build summary. 

Would be nice to have an option to show the BSN which is the normal thing to use in the Eclipse IDE. This used to the old behavior.

Example: We have a feature, and Tycho reports that bundle A is missing. The list of modules does not show A but its name which make the mapping harder for the client.

**Comment by laeubi** [comment]
I think we already has had a similar discussion before. If you have strict requirements that Name matches BSN you should simply name your bundle that way. The build-output is a bad tool to debug things. The Eclipse IDE does not show the BSN it shows the name of the project what could be different even to the actual BSN or Bundle Name.

If you have issues because you got lost with your own naming (actually the Bundle Name is supposed to provide a human understandable name), it might maybe be a better approach to have an "Open Bundle" Like (Open Type and Opeen Resource) in PDE that filters for the real Bundle Name.

**Reply by laeubi** [reply]
By the way its actually maven showing the project name in the summary not Tycho...

**Comment by vogella** [comment]
Ok, kind of makes sense. Still annoying as the old output was simply better but I understand your reasoning. And features uses the BSN not the project name nor the bundle name.

**Reply by laeubi** [reply]
Features use the feature name, try it out. The "old" output was because previously the project name was not set.

**Comment by laeubi** [comment]
@vogella has you considered [suggesting this to maven](https://issues.apache.org/jira/projects/MNG/issues/) to have a cli-switch to show artifact-ids instead of module names?
